core/src/main/scala/kafka/admin/ReassignPartitionsCommand.scala

```diff
@@ -962,22 +963,18 @@ object ReassignPartitionsCommand extends Logging {
     verifyBrokerIds(adminClient, proposedParts.values.flatten.toSet)
     val currentParts = getReplicaAssignmentForPartitions(adminClient, proposedParts.keySet.toSet)
     println(currentPartitionReplicaAssignmentToString(proposedParts, currentParts))
-    if (interBrokerThrottle >= 0 || logDirThrottle >= 0) {
+
+    if (interBrokerThrottle > 0 || logDirThrottle > 0) {
```

Comment:
It does nothing now when the value is equal to 0. Does it break compatibility (Although we had said that ``` The throttle rate should be at least 1 KB/s.```)?  By contrast, the new method ```modifyInterBrokerThrottle``` send request even if the value is 0. why?

Reply:
I'll revert this change. I thought I had noticed an inconsistency, but maybe it was just the comment in the command description.